frontend/src/images.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "./images/*" {
```

Comment:
Possibly slightly nicer to be path-agnostic here. Could be something like... `declare module "*.png"` instead. With additional module declarations for *.jpeg, *.jpg, or *.gif present in the same file, if need-be.

Reply:
Good call, changed to `declare module "*.png"`, which became instantly useful when I changed the name of the directory from `images` to `assets` 😆